src/CheckersBoard2.java

```diff
@@ -0,0 +1,227 @@
+//package goes here
+inport java.util.ArrayList;
+
+//Basic classes for checkers command line version
+
+public class CheckersBoard2 implements CheckersGame
```

Comment:
I'm not sure this should be here yet - looks like work in progress.  It won't hurt to merge it if it's not referenced anywhere, but it might make things messier to leave it in.  Your call.

Reply:
And this gets the full 50 bonus XP (this won't break, and rejecting invalid coordinates is currently handled by `findCoordinates()`.